client/src/components/MusicPlayer/MusicPlayer.tsx

```diff
@@ -8,6 +11,24 @@ import {
 } from '../../styles/MusicPlayerStyle';
 
 function MusicPlayer() {
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    const getUserInfo = async () => {
+      const response = await fetch('/auth/me');
```

Comment:
This works, but would this still work if you used axios instead? I saw you installed axios for your project.

Reply:
I mostly installed axios to test something. I'm not going to be using axios front-end for this project.